sqlserver/datadog_checks/sqlserver/activity.py

```diff
@@ -286,6 +293,10 @@ def _obfuscate_and_sanitize_row(self, row):
             )
             # sqlserver doesn't have a boolean data type so convert integer to boolean
             comments, row['is_proc'], procedure_name = extract_sql_comments_and_procedure_name(row['text'])
+            if 'tail_text' in row:
+                appended_comments, _ = extract_sql_comments(row['tail_text'])
+                if appended_comments:
+                    comments = list(set(comments + appended_comments))
```

Comment:
why do we want to dedupe comments?

Reply:
we might have captured the same comment twice, e.g. for: `select /* comment */ from table`. (Imagine this SQL is 501 chars long.)